City-of-Helsinki/employment-services-ui
# THF-593: HDS update 2.17

HDS 2.17 update 
- Updated package dependencies

https://github.com/City-of-Helsinki/helsinki-design-system/releases/tag/v2.17.0

#### Testing

- Go around the page and see that nothing has changed.
- Create example basic page and add different paragraphs to the page. 
- check that everything works.
